### PR TITLE
add RadioGroup custom input

### PR DESCRIPTION
**Pull request by TyranRexEXE**

### CODE REVIEW

File: src/connection/database/index.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ function processDatabaseOptions(opts) {
         prefill = undefined;
       }
 
-      const types = ['select', 'text', 'checkbox', 'hidden', 'textinfo'];
+      const types = ['select', 'text', 'checkbox', 'hidden', 'textinfo', 'radio_group'];
```
Comment (jagathwee) [inline src/connection/database/index.js]: to be consistent with other types: radio_group -> radiogroup

**Review by jagathwee — APPROVED** [review]
LGTM (pls check the comment)

File: src/ui/input/radio_group.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    result.forEach((item) => {
+      if (typeof item === 'object' && item.hasOwnProperty('value') && item.hasOwnProperty('label')
+      && typeof item.value === 'string' && typeof item.label === 'string') {
+        isInvalidOptions = isInvalidOptions || false
+      } else {
+        isInvalidOptions = true;
+      }
+    })
+    if (isInvalidOptions) {
+      console.warn(`Invalid options for ${name} field`)
```
Comment (jagathwee) [inline src/ui/input/radio_group.jsx:29]: Do we need this warning? Is it only for debugging?

Comment (TyranRexEXE) [reply src/ui/input/radio_group.jsx:29]: we need this warning

File: src/connection/database/index.js (hunk 1 above)
Comment (TyranRexEXE) [reply src/connection/database/index.js]: done

**Pull request merged**